Sources/MeiliSearch/Model/SearchParameters.swift

```diff
@@ -72,35 +83,60 @@ public struct SearchParameters: Codable, Equatable {
     }
 
     func dictionary() -> [String: String] {
-      var dic = [String: String]()
-      dic["q"] = query
-      dic["offset"] = "\(offset)"
-      dic["limit"] = "\(limit)"
-
-      if let attributesToRetrieve = self.attributesToRetrieve {
-        dic["attributesToRetrieve"] = commaRepresentation(attributesToRetrieve)
-      }
-
-      if !attributesToCrop.isEmpty {
-        dic["attributesToCrop"] = commaRepresentation(attributesToCrop)
-      }
-
-      dic["cropLength"] = "\(cropLength)"
-
-      if !attributesToHighlight.isEmpty {
-        dic["attributesToHighlight"] = commaRepresentation(attributesToHighlight)
-      }
-
-      if let filters: Filter = self.filters {
-        dic["filters"] = "\(filters.attribute):\(filters.value)"
-      }
-
-      dic["matches"] = "\(matches)"
-      return dic
+        var dic = [String: String]()
+        dic["q"] = query
+        dic["offset"] = "\(offset)"
+        dic["limit"] = "\(limit)"
+
+        if let attributesToRetrieve = self.attributesToRetrieve {
+            dic["attributesToRetrieve"] = commaRepresentation(attributesToRetrieve)
+        }
+
+        if !attributesToCrop.isEmpty {
+            dic["attributesToCrop"] = commaRepresentation(attributesToCrop)
+        }
+
+        dic["cropLength"] = "\(cropLength)"
+
+        if !attributesToHighlight.isEmpty {
+            dic["attributesToHighlight"] = commaRepresentation(attributesToHighlight)
+        }
+
+        if let filters: Filter = self.filters {
+            dic["filters"] = "\(filters.attribute):\(filters.value)"
```

Comment:
Are you able to do multiple filters with this?
Is this possible filter possible?

`release_date > 795484800 AND (director = "Tim Burton" OR director = "Christopher Nolan"`

It looks like this in javascript
```javascript
client.getIndex('movies').search('Batman', {
  filters: 'release_date > 795484800 AND (director = "Tim Burton" OR director = "Christopher Nolan")'
})
```

Reply:
No really, when the SDK was build there were no support for multiple filters, I am sorting this in this PR by making the Filter attribute a string.